scrape_rec/settings.py

```diff
@@ -1,94 +1,30 @@
-# -*- coding: utf-8 -*-
-
-# Scrapy settings for scrape_rec project
-#
-# For simplicity, this file contains only settings considered important or
-# commonly used. You can find more settings consulting the documentation:
-#
-#     https://doc.scrapy.org/en/latest/topics/settings.html
-#     https://doc.scrapy.org/en/latest/topics/downloader-middleware.html
-#     https://doc.scrapy.org/en/latest/topics/spider-middleware.html
-
 BOT_NAME = 'scrape_rec'
 
 SPIDER_MODULES = ['scrape_rec.spiders']
 NEWSPIDER_MODULE = 'scrape_rec.spiders'
 
-
-# Crawl responsibly by identifying yourself (and your website) on the user-agent
-#USER_AGENT = 'scrape_rec (+http://www.yourdomain.com)'
-
-# Obey robots.txt rules
 ROBOTSTXT_OBEY = True
 
-# Configure maximum concurrent requests performed by Scrapy (default: 16)
-#CONCURRENT_REQUESTS = 32
-
-# Configure a delay for requests for the same website (default: 0)
-# See https://doc.scrapy.org/en/latest/topics/settings.html#download-delay
-# See also autothrottle settings and docs
-DOWNLOAD_DELAY = 3
-
-# The download delay setting will honor only one of:
-#CONCURRENT_REQUESTS_PER_DOMAIN = 16
-#CONCURRENT_REQUESTS_PER_IP = 16
-
-# Disable cookies (enabled by default)
-#COOKIES_ENABLED = False
+DOWNLOAD_DELAY = 1
```

Comment:
What's this do?

Reply:
This is the time to wait between requests. In the back scrapy also has a setting for randomize_download_delay which ensures that the time between requests is download_delay * 0.5 <= delay <= download_delay * 1.5 so you have some variation and it's harder to get detected